pkgs/pubspec_parse/pubspec.yaml

```
@@ -1,5 +1,5 @@
 name: pubspec_parse
-version: 1.4.0
+version: 1.5.0
```

```suggestion
version: 1.5.0-wip
```

Since we're not quite ready to publish.